api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -505,96 +490,89 @@ public void testLocalize() {
     JSONObject aouJson =
-        dataUriToJson(localizeMap.get("workspaces/proj__wsid/.all_of_us_config.json"));
+        dataUriToJson(localizeMap.get("workspaces/myotherworkspace/.all_of_us_config.json"));
     assertThat(aouJson.getString("BILLING_CLOUD_PROJECT")).isEqualTo("other-proj");
   }
 
   @Test
   public void testLocalize_noNotebooks() {
     ClusterLocalizeRequest req = new ClusterLocalizeRequest();
-    req.setWorkspaceNamespace(WORKSPACE_NS);
-    req.setWorkspaceId(WORKSPACE_ID);
     req.setPlaygroundMode(false);
     stubGetWorkspace(WORKSPACE_NS, WORKSPACE_ID, LOGGED_IN_USER_EMAIL);
-    ClusterLocalizeResponse resp =
-        clusterController.localize(BILLING_PROJECT_ID, "cluster", req).getBody();
-    verify(notebookService).localize(eq(BILLING_PROJECT_ID), eq("cluster"), mapCaptor.capture());
+    ClusterLocalizeResponse resp = clusterController.localize(BILLING_PROJECT_ID, req).getBody();
+    verify(notebookService)
+        .localize(eq(BILLING_PROJECT_ID), eq(getClusterName()), mapCaptor.capture());
 
     // Config files only.
     Map<String, String> localizeMap = mapCaptor.getValue();
     assertThat(localizeMap.keySet())
         .containsExactly(
-            "workspaces_playground/wsid/.all_of_us_config.json",
-            "workspaces/wsid/.all_of_us_config.json");
-    assertThat(resp.getClusterLocalDirectory()).isEqualTo("workspaces/wsid");
+            "workspaces_playground/myfirstworkspace/.all_of_us_config.json",
+            "workspaces/myfirstworkspace/.all_of_us_config.json");
+    assertThat(resp.getClusterLocalDirectory()).isEqualTo("workspaces/myfirstworkspace");
   }
 
   @Test
-  public void listCluster_validateActiveBilling() {
+  public void GetCluster_validateActiveBilling() {
     doThrow(ForbiddenException.class)
         .when(workspaceService)
         .validateActiveBilling(WORKSPACE_NS, WORKSPACE_ID);
 
-    assertThrows(
-        ForbiddenException.class, () -> clusterController.listClusters(WORKSPACE_NS, WORKSPACE_ID));
+    assertThrows(ForbiddenException.class, () -> clusterController.getCluster(WORKSPACE_NS));
   }
 
   @Test
-  public void listCluster_validateActiveBilling_checkAccessFirst() {
+  public void getCluster_validateActiveBilling_checkAccessFirst() {
     doThrow(ForbiddenException.class)
         .when(workspaceService)
         .validateActiveBilling(WORKSPACE_NS, WORKSPACE_ID);
 
     doThrow(ForbiddenException.class)
         .when(workspaceService)
-        .enforceWorkspaceAccessLevel(WORKSPACE_NS, WORKSPACE_ID, WorkspaceAccessLevel.READER);
+        .enforceWorkspaceAccessLevel(WORKSPACE_NS, WORKSPACE_ID, WorkspaceAccessLevel.WRITER);
 
-    assertThrows(
-        ForbiddenException.class, () -> clusterController.listClusters(WORKSPACE_NS, WORKSPACE_ID));
+    assertThrows(ForbiddenException.class, () -> clusterController.getCluster(WORKSPACE_NS));
     verify(workspaceService, never()).validateActiveBilling(anyString(), anyString());
```

Comment:
Do we still need the first `doThrow` if we're going to verify that the method that does the throwing isn't called? Here and below.